### PR TITLE
Add 7.17.7 Fleet/Agent release notes

### LINKED ISSUE #2258: [Request] Add Fleet/Agent release notes for 7.17.7
## Description

Need to add release notes for the upcoming 7.17.7 release.

## Collaboration
- The docs team will lead producing the content

### PR DESCRIPTION
**Pull request by dedemorton**
Closes #2258

### CODE REVIEW

**Comment by juliaElastic** [comment]
the title says 8.5, should it be 7.17?

**Comment by dedemorton** [comment]
@juliaElastic Yup! My brain was stuck in the 8.5 groove I guess?

**Review by juliaElastic — APPROVED** [review]
Fleet UI LGTM, didn't see any change for 7.17.7

File: docs/en/ingest-management/release-notes/release-notes-7.17.asciidoc [hunk 1]
```diff
@@ -33,6 +35,48 @@ Also see:
...
+[[breaking-changes-7.17.7]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and
+performance. Before you upgrade, review the breaking changes, then mitigate the
+impact to your application.
+
+[discrete]
+[[breaking-PR32493]]
+.{fleet-server} now rejects certificates signed with SHA-1
```
Comment (cmacknz) [inline docs/en/ingest-management/release-notes/release-notes-7.17.asciidoc:55]: This applies to agent as well as Fleet server.

Comment (cmacknz) [reply docs/en/ingest-management/release-notes/release-notes-7.17.asciidoc:55]: Actually it appears we did not backport this change to 7.17 in fleet-server, so this only applies to agent in 7.17.

Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-7.17.asciidoc:55]:
```suggestion
.{elastic-agent} now rejects certificates signed with SHA-1
```

**Review by cmacknz — APPROVED** [review]

**Pull request merged**